generators/entity/index.js

```diff
@@ -116,7 +116,15 @@ module.exports = class extends BaseGenerator {
           this.log(`\n${chalk.bold.green('I\'m updating the entity for audit ')}${chalk.bold.yellow(this.entityConfig.entityClass)}`);
 
           const entityName = this.entityConfig.entityClass;
-          const jsonObj = this.entityConfig.data;
+          var jsonObj = {};
+          if(this.entityConfig.data === undefined) {
```

Comment:
space after if, (please run `npm lint`)

Reply:
I install eslint to vs code and check, thanks.